tests/integration/acp/add_policy/with_no_resources_test.go

```diff
@@ -77,9 +77,10 @@ func TestACP_AddPolicy_NoResourceLabel_ValidID(t *testing.T) {
 // A Policy can have no resources (incompatible with DPI) but it needs a name.
 func TestACP_AddPolicy_PolicyWithOnlySpace_NameIsRequired(t *testing.T) {
 	test := testUtils.TestCase{
-
 		Description: "Test acp, adding a policy that has only space",
-
+		SupportedACPTypes: immutable.Some([]testUtils.ACPType{
```

Comment:
suggestion: conditionally skip these tests if using SoruceHub ACP for now. The empty name validation is implemented in ACP Core but not on sourcehub yet. Once SourceHub consumes ACP core it will work as intended

Reply:
> suggestion: conditionally skip these tests if using SoruceHub ACP for now.

That is what `SupportedACPTypes` does :)